src/main/java/tech/ydb/io/r2dbc/context/YdbConst.java

```diff
@@ -0,0 +1,11 @@
+package tech.ydb.io.r2dbc.context;
+
+/**
+ * @author kuleshovegor
+ */
+public final class YdbConst {
+    public static final String MISSING_VALUE_FOR_PARAMETER = "Missing value for parameter: ";
```

Comment:
Do we really want to store errors as constants?

Reply:
Used this message local